viewer-prototype/src/browser/trace-server-status.ts

```diff
@@ -120,11 +122,21 @@ export class TraceServerConnectionStatusContribution extends DefaultFrontendAppl
     }
 
     private async startServer() {
-        await this.traceServerConfigService.startTraceServer(this.path, this.port);
+        // TODO: Add proper error message for the port as well. Right now the server doesn't return any error code or message whenever a port is incorrect and/or the port is busy.
```

Comment:
What happens if the port is incorrect or busy at the moment? Is there really nothing that can be leveraged?

Reply:
@marechal-p the latest trace server from today will exit with a non-zero return value which should be checked in trace-server-service.ts

Reply:
> What happens if the port is incorrect or busy at the moment? Is there really nothing that can be leveraged?

Earlier it couldn't be done however with bernd's latest server patch we should be able to leverage the exit code that is returned from the trace server.

Reply:
Done.